lab_3/main.py

```diff
@@ -31,38 +62,62 @@ class LetterStorage:
 
     def __init__(self):
         self.storage = {}
+        self.unique_id = 0
 
     def _put_letter(self, letter: str) -> int:
         """
         Puts a letter into storage, assigns a unique id
         :param letter: a letter
         :return: 0 if succeeds, 1 if not
         """
-        pass
+        if not isinstance(letter, str) or not letter:
+            return -1
+
+        if letter not in self.storage:
+            self.storage[letter] = self.unique_id
+            self.unique_id += 1
```

Comment:
can you assign ids without adding additional self.unique_id parameter?

Reply:
okay, i'll rewrite this part

Reply:
I think I can use len() in order to avoid unique_id
self.storage[letter] = len(self.storage) + 1